### ISSUE TITLE
📎 Human readable timestamp in recover list

### ISSUE THREAD

**Issue by jer3m01** [rome/tools#1660]
### Description

Timestamps displayed on `rome recover list` are hard to read:
![image](https://user-images.githubusercontent.com/33844379/126882212-cd0df2b7-ca4b-4457-847b-754f30c5df93.png)

We should at least show a human readable version next to it or just human & timestamp.
```
1597569118661-check-0 ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

Ran 11 months, 1 week, 17 hours, 25 minutes and 50.98 seconds ago (2020-08-16T09:11:58.661Z)
```

**Comment by imskr** [comment]
Can I work on this @jer3m01 ?

**Comment by jer3m01** [comment]
> 
> 
> Can I work on this @jer3m01 ?

@imskr Sure! The formatter already exists: https://github.com/rome/tools/blob/3781e838239c50f3a37baf87fa312bfdb5cd4eec/internal/numbers/Duration.ts#L209-L242

Just need to find where the output is from, enable `longform` and maybe add it to our markup tag (https://github.com/rome/tools/tree/main/internal/markup#duration)

**Comment by imskr** [comment]
Thanks @jer3m01

**Comment by jamiebuilds** [comment]
Hi! Thank you for opening this issue

We recently made the decision to [migrate the Rome codebase to Rust](https://rome.tools/blog/2021/09/21/rome-will-be-rewritten-in-rust) and making some significant changes to its architecture. We've also decided not to release the previous version written in TypeScript.

We're going through and closing pull requests and issues that were specific to the TypeScript codebase, including this one. Thank you for taking the time to open it.

Thanks again!
